manifests/plugin/write_kafka.pp

```diff
@@ -4,9 +4,11 @@
   Array[String] $kafka_hosts = ['localhost:9092'],
   $kafka_port                = 9092,
   Hash $topics               = {},
+  Hash $properties           = {},
+  Hash $meta                 = {},
 ) {
 
-  include collectd
+  include ::collectd
```

Comment:
Please don't add the topscope to the include. That's a deprecated puppet 3 syntax.